src/i18n/locale/en.json

```diff
@@ -369,21 +369,21 @@
     },
     "backup": {
       "label": {
-        "backup_restore_failed": "Backup restore failed!",
+        "backup_restore_failed": "Could not restore backup",
         "create_backup": "Create Backup",
-        "create_backup_info": "Backup library as a Tachiyomi backup",
+        "create_backup_info": "Back up library as a Tachiyomi backup",
         "legacy_backup_unsupported": "legacy backups are not supported!",
         "restore_backup": "Restore Backup",
-        "restore_backup_info": "You can also drag and drop the backup file here to restore",
-        "restored_backup": "Backup restore finished!",
-        "restoring_backup": "Restoring backup..."
+        "restore_backup_info": "You can also drag and drop the backup file here to restore it",
+        "restored_backup": "Backup restored.",
+        "restoring_backup": "Restoring backup…"
       },
       "title": "Backup"
     },
     "label": {
       "dark_theme": "Dark Theme",
```

Comment:
```suggestion
      "dark_theme": "Dark theme",
```

Reply:
proper noun, and typography consideration